src/cloco-client.ts

```diff
@@ -150,12 +151,27 @@ export class ClocoClient {
     private async initializeApplication(): Promise<void> {
 
       Logger.log.debug(`ClocoClient.initializeApplication: start.`);
+      let filename: string = `~/.cloco/cache/application_${this.options.application}`;
```

Comment:
The idea is that your cloco application maps 1:1 with your actual nodejs application.  Therefore you pass into the options the name of the subscription and the application and the client will load up all of the configuration for the application and keep it refreshed.

You could have >1 node client running on the same machine but I don't envisage *at the moment* having >1 running in the same nodejs application